oxidation/client/src/theme/variables.css

```
@@ -4,15 +4,13 @@ http://ionicframework.com/docs/theming/ */
 @font-face {
   font-family: 'montserrat';
   font-style: normal;
-  font-weight: normal;
-  src: url('../assets/fonts/montserrat.ttf');
+  src: url('../assets/fonts/montserrat-VariableFont_wght.ttf');
```

I would suggest to rename the file 

```suggestion
url('../assets/fonts/montserrat-VariableFont_weight.ttf');
```

---

The name was suggest by Google when I downloaded the font, I simply remove "wght".